tools/publisher/src/subcommand/publish.rs

```diff
@@ -6,8 +6,8 @@
 use crate::cargo::{self, CargoOperation};
 use crate::fs::Fs;
 use crate::package::{
-    continue_batches_from, discover_package_batches, Package, PackageBatch, PackageHandle,
-    PackageStats,
+    continue_batches_from, discover_and_validate_package_batches, Package, PackageBatch,
```

Comment:
imo we should remove continue_batches_from—it doesn't actually work and could lead to use failing to publish packages.

Reply:
I'm onboard with this. Will remove.